apps/rotatelogos/templates/rotatelogos/index.html

```diff
@@ -0,0 +1 @@
+<img src="{{img_url}}"></img>
```

Comment:
<img> has no end tag, but mandatory alt property. Consider changing it to <img src="{{img_url}}" alt="{{img_name}}">

Also, you make git happier by adding a newline at the end of a file :)